core/libp2p/network/impl/connection_manager_impl.cpp

```diff
@@ -0,0 +1,120 @@
+/**
+ * Copyright Soramitsu Co., Ltd. All Rights Reserved.
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+#include "connection_manager_impl.hpp"
+
+#include <algorithm>
+
+namespace libp2p::network {
+
+  std::vector<ConnectionManager::ConnectionSPtr>
```

Comment:
The method can return a span

Reply:
No, it can not. Because underlying storage is a "map", and we actually need to allocate new vector, and copy all connections from all "values" in map.

Reply:
This method returns all vectors